healthScore/urls.py

```diff
@@ -5,12 +5,38 @@
 
 urlpatterns = [
     path("homepage", views.homepage, name="homepage"),
-    path("registration", views.registration, name="registration"),
-    path("viewHealthHistory", views.view_health_history, name="view_health_history"),
-    path("viewReports", views.view_report, name="view_reports"),
-    path("login", views.login_view, name="login"),
-    path("userInfo", views.view_user_info, name="user_info"),
-    path("viewRequests", views.view_health_history_requests, name="view_requests"),
+    path("registration/", views.registration, name="registration"),
+    path("login/", views.login_view, name="login"),
     path("logout/", auth_views.LogoutView.as_view(), name="logout"),
+    path(
+        "password-reset/",
```

Comment:
Since this is a url, could we have the path in camelCase to keep all urls uniform? The same for the password-reset-done url as well